### PR TITLE
[PUBLISH] [CONTENT_API] moved code handling content_api packages to p…

### PR DESCRIPTION
**Pull request by jerome-poisson**
…ublish_package

SDESK-2073

### CODE REVIEW

**Comment by petrjasek** [comment]
would be good to add some test for this

File: apps/publish/enqueue/enqueue_service.py [hunk 1]
```diff
@@ -345,6 +345,13 @@ def publish_package(self, package, target_subscribers):
             if temp_queued:
                 queued = temp_queued
 
+            delivery_types = [d['delivery_type'] for d in self.get_destinations(subscriber)]
+            is_content_api_delivery = 'content_api' in delivery_types
```
Comment (petrjasek) [inline apps/publish/enqueue/enqueue_service.py]: afaik there is no content_api delivery type

Comment (jerome-poisson) [reply apps/publish/enqueue/enqueue_service.py]: @petrjasek it's used in several places (e.g. [there](https://github.com/superdesk/superdesk-core/blob/master/superdesk/publish/publish_queue.py#L146) and [there](https://github.com/superdesk/superdesk-core/blob/master/apps/publish/enqueue/enqueue_service.py#L499)

It is set [there](https://github.com/superdesk/superdesk-core/blob/master/apps/publish/enqueue/enqueue_service.py#L353) and the code is working anyway.

Comment (petrjasek) [reply apps/publish/enqueue/enqueue_service.py]: I see, thx

**Pull request merged**